docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '3.9'
```

Comment:
Interesting the docker website doesn't state there is a `3.9`: https://docs.docker.com/compose/compose-file/compose-versioning/